packages/js/src/workouts/components/NewsletterSignup.js

```diff
@@ -105,8 +107,8 @@ export function NewsletterSignup() {
 								"<a>",
 								"</a>"
 							),
-							"https://yoa.st/gdpr-config-workout",
-							"yoast-configuration-workout-gdpr-link"
+							gdprLink,
+							"yoast-configuration-workout-gdpr-gdprLink"
```

Comment:
Not sure why we want to change that (and mention `gdpr` twice)?

Reply:
Right, the second one should not have been changed! Good catch.